### ISSUE TITLE
[API] Allow for MI Integration

### ISSUE THREAD

**Issue by Sayshal** [kgar/foundry-vtt-tidy-5e-sheets#779]
Topic is more broad than title, but I'm looking to allow Magic Items' spells to be sortable via the Configure Sections menu.
![vtt-desktop-client_wBGZGW0pCU](https://github.com/user-attachments/assets/7bb29447-2e40-45e0-b367-173c33ab642f)

[MI Issue Here](https://github.com/PwQt/magic-items-2/issues/181)

**Comment by Sayshal** [comment]
Can probably close this with 4.x replacing MI. @kgar

**Comment by kgar** [comment]
Yeah.... It seems that way. At the very least, the effort to make MI integrate at that level is a lot of time and effort, and having the overhaul work to do and all the Foundry / System updates to account for keeps pushing it back.

I'll close it out as unplanned. Sorry I couldn't get this one.
